### ISSUE TITLE
setup CI Jenkins

### ISSUE THREAD

**Issue by brenzi** [scs/substrate-api-client#25]
Please set up CI for this crate

sanity checks:
```
cargo build --release
cargo build --release --no-default-features
cargo build --examples
cargo test
```
run examples against substrate-test-node (master?) (possible with jenkins?)

**Comment by brenzi** [comment]
please also test no_std build with 
```
cd test_no_std
cargo build
```
The problem is that the linker still fails for this crate....

**Comment by brenzi** [comment]
update: @clang fixed the linker issue in 5e7524f646c6755f741a1f06ac3e87b23956892d

**Comment by electronix** [comment]
Mostly done.

Implemented:
* Build every night a substrate-test-node (master)
* Build every night all targets (except test_no_std) of substrate-api-client
* Run examples (except `contract` and `event_callback` because of timeout) against substrate-test-node
* Cargo warning collection and visualization
* Mail notification if unstable

Remaining points are handled in #33
